iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings

```diff
@@ -1572,7 +1572,7 @@
 
 "type.barrier.chain" = "Cadena";
 
-"type.barrier.city_wall" = "Muro de la ciudad";
+"type.barrier.city_wall" = "City Wall";
```

Comment:
These changes are coming from some bug in our regen script sometimes. It would be great to find and fix it...
Can you please remove these changes from a commit??